content/dstu2/contract.md

```diff
@@ -42,11 +42,11 @@ Notes:
 ### Response
 
 Search for all Contracts subject to a specific patient
-<%= headers 200, GET: '[...]/Contract?subject:Patient=14067775' %>
+<%= headers 200, GET: '[...]/Contract?subject:Patient=4598008' %>
```

Comment:
The g7 Resources were validated and stood up in FHIRPLAY. I don't know if they will work in SNDENG at all and there has not been build been done in that domain.

Additionally, I question the need for these changes altogether. Historically we have specifically avoided using valid data/ids in the responses because they could be linked to actual people with actual details. If we just "fix" the URLs to work then the results returned when someone calls the service will not match the JSON examples and start giving out data that don't want to give out.

I feel like the correct way to address this is to have custom build data configured specifically for these queries. Of course, good luck policing that when it comes to writes (see: DocumentReference)